src/main/java/org/skriptlang/skript/bukkit/spark/expressions/ExprMSPT.java

```diff
@@ -0,0 +1,89 @@
+package org.skriptlang.skript.bukkit.spark.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import me.lucko.spark.api.statistic.StatisticWindow.MillisPerTick;
+import me.lucko.spark.api.statistic.misc.DoubleAverageInfo;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.spark.MsptStatistic;
+import org.skriptlang.skript.bukkit.spark.SparkUtils;
+
+import java.util.Arrays;
+
+@Name("MSPT Usage")
+@RequiredPlugins("Paper 1.21+ or Spark")
+@Description({
+    "Returns the MSPT (milliseconds per tick) readings from the last 10 seconds, 1 minute and 5 minutes. " +
+	"The returned values will be numbers which represent milliseconds, rather than timespans.",
+	"The average, minimum, maximum, and median values are available, as well as any percentile between 0 and 100.",
+    "This expression can only be used if the server has Spark or you have Spark installed as a plugin."
```

Comment:
```suggestion
    "Returns the MSPT (milliseconds per tick) readings from the last 10 seconds, 1 minute and 5 minutes. " +
    "The returned values will be numbers which represent milliseconds, rather than timespans.",
    "The average, minimum, maximum, and median values are available, as well as any percentile between 0 and 100.",
    "This expression can only be used if the server has Spark or you have Spark installed as a plugin."
```

Reply:
Are these spaces? There shouldn't be any